src/grandorgue/gui/GOGUILabel.cpp

```diff
@@ -202,6 +207,22 @@ void GOGUILabel::Load(GOConfigReader &cfg, wxString group) {
   InitBackgroundBitmap(
     x,
     y,
+    cfg.ReadInteger(
+      ODFSetting,
+      group,
+      wxT("Width"),
+      1,
+      m_metrics->GetScreenWidth(),
+      false,
+      0),
+    cfg.ReadInteger(
+      ODFSetting,
+      group,
+      wxT("Height"),
+      1,
+      m_metrics->GetScreenHeight(),
+      false,
+      0),
     cfg.ReadStringTrim(ODFSetting, group, WX_IMAGE, false),
     cfg.ReadInteger(ODFSetting, group, WX_DISP_IMAGE_NUM, 0, 12, false, 1),
```

Comment:
Could you please also change this line to allow up to image 15 instead of only 12?

Reply:
Done